.github/workflows/tests-ci.yml

```diff
@@ -125,17 +125,10 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
         with:
           submodules: true
-      
-      - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d #v3.0.0
-        with:
-          version: 8
 
-      - name: Set up Node.js
-        uses: actions/setup-node@5ef044f9d09786428e6e895be6be17937becee3a #v4.0.0
-        with:
-          node-version: 18
-          registry-url: https://registry.npmjs.org/
-          cache: 'pnpm'
+      # https://cashapp.github.io/hermit/usage/ci/
+      - name: Init Hermit
+        uses: cashapp/activate-hermit@v1
```

Comment:
@phoebe-lew @shamilovtim Does caching still work with this approach?  Any impact to CI total runtime?

Reply:
The cashapp/activate-hermit action does also support a `cache: true` attribute FYI.

Reply:
should that param be passed here?

Reply:
I see no difference in CI time though

Reply:
@shamilovtim will add!